packages/create-plugin/templates/datasource/src/datasource.ts

```diff
@@ -77,7 +76,7 @@ export class DataSource extends DataSourceApi<MyQuery, MyDataSourceOptions> {
       }
     } catch (err) {
       let message = '';
-      if (_.isString(err)) {
+      if (typeof err === 'string') {
```

Comment:
I couldn't find any other places we are using `lodash` on scaffolding so I removed it - the developers can add it back when needed and it is removed by webpack on bundle anyway.

Reply:
@jackw anything against that ☝️

Reply:
👍 

This method is a bit of a mess, so would be nice to improve at some point. Maybe grafana/data could expose a function that returns message from an unknown error.